libs/elements/layout/footer/src/lib/components/footer/footer.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+
+// socials urls
+const ig = "https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690295/elewa-group-website/Icons/SVG/_Social/instagram_x8ujcq.svg"
+const linkedIn = "https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690294/elewa-group-website/Icons/SVG/_Social/linkedin_x8smhm.svg"
+const fb ="https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690294/elewa-group-website/Icons/SVG/_Social/facebook_lhhwk2.svg"
```

Comment:
Quick note: In future, always make sure you remove code that helped you in testing during development.

Reply:
@Beulah-Matt I think the variables above were okay since he's using them in the template a tags just wrong file placement. Next time you can create a utils file in your module that exports the links above @jbdura